tests/load/bigquery/test_bigquery_table_builder.py

```diff
@@ -213,7 +217,7 @@ def test_create_table_with_partition_and_cluster(gcp_client: BigQueryClient) ->
     sqlfluff.parse(sql, dialect="bigquery")
     # clustering must be the last
     assert sql.endswith("CLUSTER BY `col2`, `col5`")
-    assert "PARTITION BY `col10`" in sql
+    assert "PARTITION BY `col10`" in sql or "PARTITION BY col10" in sql
```

Comment:
@rudolfix do you prefer to just remove `col10`? By default, sqlglot does not quote the column name if it is not needed e.g. names with space